brokenspoke_analyzer/core/database/dbcore.py

```diff
@@ -12,6 +12,12 @@
 from brokenspoke_analyzer.core import runner
 
 
+def execute_query_with_result(engine: Engine, query: str) -> typing.Any:
```

Comment:
I would specify the full return type here: `typing.Sequence[Row[typing.Any]]`.

Reply:
Ah, yep, I meant to follow up and fix that. Should be fixed now.